crates/re_viewer/src/app.rs

```diff
@@ -70,6 +70,9 @@ pub struct StartupOptions {
 
     /// Forces wgpu backend to use the specified graphics API.
     pub force_wgpu_backend: Option<String>,
+
+    #[cfg(target_arch = "wasm32")]
+    pub panel_state_overrides: PanelStateOverrides,
```

Comment:
Seems simpler to include this on native too. Even though we don't currently support setting it, we might want to in the future using CLI flags or env-vars.